CMakeLists.txt

```diff
@@ -67,6 +67,10 @@ option(BUILD_CPP_ENABLE_METRICS
     "Enable the metrics collection at compile phase"
     OFF)
 
+option(BUILD_ENABLE_AVX512
+    "Enable AVX512 vector decode of bit-packing"
+    OFF)
```

Comment:
Well, this does mean we are going to skip GitHub Action testing on this PR, @wpleonardo ?

Reply:
No, I just wanna to check the CI test result when disable AVX512 feature, because I don't have these different platforms and my own orc_test ruuning success on local platorm(centos 8).
In the end, this option will be set as "ON" follow your above suggestions. Thanks.